freestyle-logging/jvm/src/main/scala/io/freestyle/loggingJVM.scala

```diff
@@ -0,0 +1,32 @@
+package io.freestyle
+
+import cats.MonadError
+import io.freestyle.logging._
+import journal._
+
+object loggingJVM {
+
+  object implicits {
+    implicit def freeStyleLoggingInterpreter[M[_], C: Manifest](implicit ME: MonadError[M, Throwable]): LoggingM.Interpreter[M] =
+      new LoggingM.Interpreter[M] {
+
+        val logger = Logger[C]
+
+        def debugImpl(msg: String): M[Unit] = ME.catchNonFatal(logger.debug(msg))
```

Comment:
Trying to understand why we need a MonadError instance to wrap all these calls. Also it may be nice to have the rationale added to the interpreter for future reference.